pingpong/video_store.py

```
@@ -189,6 +220,36 @@ def _resolve_key_path(self, key: str) -> Path:
 
+    def _write_file_in_chunks(self, file_path: Path, content: IO) -> None:
+        with open(file_path, "wb") as handle:
+            while True:
+                chunk = content.read(self._WRITE_CHUNK_SIZE)
+                if not chunk:
+                    break
+                if isinstance(chunk, str):
+                    chunk = chunk.encode("utf-8")
+                handle.write(chunk)
+
+    async def put(self, key: str, content: IO, content_type: str):
```

Ruff (0.15.5)

[warning] Unused method argument: `content_type`

(ARG002)